docs/resources/playbook.md

```diff
@@ -0,0 +1,69 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "ansible_playbook Resource - terraform-provider-ansible"
+subcategory: ""
+description: |-
+  
+---
+
+# ansible_playbook (Resource)
+
+Provides an Ansible playbook resource.
+
+## Example Usage
+```terraform
+resource "ansible_playbook" "playbook" {
+  playbook   = "playbook.yml"
+  name       = "host-1.example.com"
+  replayable = true
+
+  extra_vars = {
+    var_a = "Some variable"
+    var_b = "Another variable"
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) Name of the desired host on which the playbook will be executed.
+- `playbook` (String) Path to ansible playbook.
+
+### Optional
+
+- `ansible_playbook_binary` (String) Path to ansible-playbook executable (binary).
+- `check_mode` (Boolean) If 'true', playbook execution won't make any changes but only change predictions will be made.
+- `diff_mode` (Boolean) If 'true', when changing (small) files and templates, differences in those files will be shown.Recommended usage with 'check_mode'.
+- `extra_vars` (Map of String) A map of additional variables as: { key-1 = value-1, key-2 = value-2, ... }.
+- `force_handlers` (Boolean) If 'true', run handlers even if a task fails.
+- `groups` (List of String) List of desired groups of hosts on which the playbook will be executed.
+- `ignore_playbook_failure` (Boolean) This parameter is good for testing.Set to 'true' if the desired playbook is meant to fail, but still want the resource to run successfully.
+- `limit` (List of String) List of hosts to exclude from the playbook execution.
+- `replayable` (Boolean) If 'true', the playbook will be executed on every 'terraform apply' and with that, the resource will be recreated.If 'false', the playbook will be executed only on the first 'terraform apply'.Note, that if set to 'true', when doing 'terraform destroy', it might not show in the destroy output, even though the resource still gets destroyed.
```

Comment:
```suggestion
- `replayable` (Boolean) If 'true', the playbook will be executed on every 'terraform apply' and with that, the resource will be recreated. If 'false', the playbook will be executed only on the first 'terraform apply'. Note, that if set to 'true', when doing 'terraform destroy', it might not show in the destroy output, even though the resource still gets destroyed.
```